servers/parser-graphql-wrapper/src/config/index.ts

```diff
@@ -62,78 +62,78 @@ export default {
     characterMap: process.env.CHARACTER_MAP
       ? new Map<string, number>(JSON.parse(process.env.CHARACTER_MAP))
       : new Map<string, number>([
-          ['G', 0],
-          ['Q', 0],
-          ['k', 0],
-          ['V', 0],
-          ['X', 0],
-          ['I', 0],
-          ['W', 1],
-          ['Y', 1],
-          ['a', 1],
-          ['j', 1],
-          ['Z', 1],
-          ['o', 1],
-          ['C', 2],
-          ['D', 2],
-          ['Q', 2],
-          ['U', 2],
-          ['Z', 2],
-          ['d', 3],
-          ['C', 3],
-          ['f', 3],
-          ['g', 3],
-          ['w', 3],
+          ['A', 0],
+          ['B', 0],
+          ['C', 0],
+          ['D', 0],
+          ['E', 0],
+          ['y', 0],
+          ['F', 1],
+          ['G', 1],
+          ['H', 1],
+          ['I', 1],
+          ['J', 1],
+          ['z', 1],
+          ['K', 2],
+          ['L', 2],
+          ['M', 2],
+          ['N', 2],
+          ['O', 2],
+          ['P', 3],
+          ['Q', 3],
+          ['R', 3],
+          ['S', 3],
+          ['T', 3],
+          ['U', 4],
           ['V', 4],
-          ['k', 4],
-          ['n', 4],
-          ['J', 4],
-          ['p', 4],
-          ['H', 5],
-          ['Q', 5],
-          ['R', 5],
-          ['t', 5],
-          ['x', 5],
-          ['w', 6],
-          ['K', 6],
-          ['J', 6],
-          ['P', 6],
-          ['G', 6],
-          ['U', 7],
-          ['q', 7],
-          ['Y', 7],
-          ['V', 7],
-          ['W', 7],
-          ['H', 8],
-          ['M', 8],
-          ['O', 8],
-          ['N', 8],
-          ['I', 8],
-          ['Y', 9],
-          ['i', 9],
-          ['o', 9],
-          ['Q', 9],
-          ['l', 9],
+          ['W', 4],
+          ['X', 4],
+          ['Y', 4],
+          ['Z', 5],
+          ['a', 5],
+          ['b', 5],
+          ['c', 5],
+          ['d', 5],
+          ['e', 6],
+          ['f', 6],
+          ['g', 6],
+          ['h', 6],
+          ['i', 6],
+          ['j', 7],
+          ['k', 7],
+          ['l', 7],
+          ['m', 7],
+          ['n', 7],
+          ['o', 8],
+          ['p', 8],
+          ['q', 8],
+          ['r', 8],
+          ['s', 8],
+          ['t', 9],
+          ['u', 9],
+          ['v', 9],
+          ['w', 9],
+          ['x', 9],
         ]),
     positionMap: process.env.POSITION_MAP
       ? new Map<number, number>(JSON.parse(process.env.POSITION_MAP))
       : new Map<number, number>([
           [9, 0],
-          [123, 2],
+          [12, 1],
```

Comment:
Numbers must be unique and < 60 on the left side. and right side must have 0-15 defined.